inc/admin/options.php

```diff
@@ -741,6 +746,26 @@ function rocket_settings_callback( $inputs ) {
 	}
 
 	/*
+	 * Option : critical CSS content for async CSS
```

Comment:
We still need the field to input the critical css, I only removed the automatic generation of it